status-server/src/public/js/client.js

```diff
@@ -42,15 +42,77 @@ function init() {
     }
   }
   // populate the LEDs with some initial colors
-  var colors = ['#ffff00', '#ffff00', '#00ffff', '#00ff07', '#772cb6'];
+  var colors = ['#ff00ff', '#ffff00', '#00ffff', '#00ff07', '#772cb6'];
   colors.forEach((color, idx) => {
     var node = document.getElementById("led" + idx);
     var ctx = node.getContext("2d");
     ctx.fillStyle = color;
     ctx.fillRect(0,0,300,300);
+    ctx.font = "10px Arial";
+    ctx.strokeText(idx ,node.width/2,node.height/2);
+
+
   });
 }
+function animate(ct){
+  var fetchResponse = window.fetch('http://localhost:3000/status'+ct+'.json'+Date.now()).then(
```

Comment:
Do I understand this right? The purpose of this function is to simulate a "click" or "tap" on the LED/button, and so update our status. The updated status will result in visible change in the LED strip. 

If that is the case, this window.fetch call needs to be a POST (it defaults to GET) with a status value. It might be useful to know about the success/failure of this request, but otherwise for now we don't need to handle anything in the callbacks as the updated value will soon show up in the continual polling we do.